src/consoleappender.cxx

```diff
@@ -27,6 +27,7 @@
 #include <log4cplus/spi/loggingevent.h>
 #include <log4cplus/thread/syncprims-pub-impl.h>
 #include <ostream>
+#include "log4cplus/spi/factory.h"
```

Comment:
Use `<log4cplus/spi/factory.h>`, not `""`. Reorder together with other log4cplus headers.

Reply:
Did I just make a mistake?  I am not familiar with github, these changes only my test when i read the source code. What i wanted to do is to pull the new changes from mater to my branch, not to push my changes to source code! Please ignore the request of mine!